llama_index/graph_stores/simple.py

```diff
@@ -46,7 +46,7 @@ def _get_rel_map(self, subj: str, depth: int = 2) -> List[List[str]]:
         rel_map = []
         if subj in self.graph_dict:
             for rel, obj in self.graph_dict[subj]:
-                rel_map.append([rel, obj])
+                rel_map.append([subj, rel, obj])
```

Comment:
Does this change how other graph stores should be building their rel map? (i.e. neo4j, kuzu, falkordb)

Reply:
Yes, previously only simple impl. was not duplicated, the cypher based ones are all with subjects in rels, too(I checked neo4j.)

But I'll double check tomorrow, too.

Reply:
- [ ] neo4j, already with subject
- [ ] kuzu, aha, it's w/o subject, added back
- [ ] falkordb/redisgraph, already with subject

Reply:
Amazing, thanks for checking!